sml_small/utils/pandas_example.py

```diff
@@ -0,0 +1,86 @@
+"""
+Example code that uses the pandas wrapper with the Totals and Components method.
```

Comment:
Can you tweak the comment to show it is generic, you can reference totals and components as an example method that can use it if you like

Reply:
Amended